cogs/mod.py

```diff
@@ -53,6 +55,66 @@ async def pm(self, ctx, user: discord.User, *, message):
         await channel_to_forward.send(msg)
         await ctx.message.delete()
 
+    @commands.command()
+    @is_moderator()
+    async def initiate_crabbo(self, ctx):
+        """Initiates secret crabbo ceremony"""
+
+        conn = sqlite3.connect(self.bot.config.db_path)
+        c = conn.cursor()
+        c.execute("SELECT Value FROM Settings WHERE Key = ?",
+                  ("CrabboMsgID", ))
+        check_msg = c.fetchone()
+        if check_msg:
+            await ctx.send("secret crabbo has already been started.")
+            conn.close()
+            return
+        crabbo_msg = await ctx.send(
+            "🦀🦀🦀 crabbo time 🦀🦀🦀\n<@&"
+            f"{discord.utils.get(ctx.guild.roles, name=self.bot.config.crabbo_role).id}"
+            "> react to this message with 🦀 to enter the secret crabbo festival\n"
+            "🦀🦀🦀 crabbo time 🦀🦀🦀")
+        c.execute("REPLACE INTO Settings VALUES (?, ?)",
```

Comment:
irrelevant: do we often perform replace and delete on the db? even if it's for settings I'm not a big fan of it